pom.xml

```diff
@@ -148,6 +153,23 @@ THE SOFTWARE.
         <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
     </scm>
     -->
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jenkins-ci.tools</groupId>
+                <artifactId>maven-hpi-plugin</artifactId>
+                <configuration>
+                    <maskClasses>
```

Comment:
Can you leave a detailed comment on why?

Reply:
@afalko Ill leave a comment here, because I had to undo the masking. The two classes are present inside Jenkins core, and the class loader loads them, however they have an optional dependency on Jettison, which core does not provide. This creates a problem that even if we provide Jettison via plugin, it does not appear in the classpath at the time Jenkins core loads it. The idea was to mask those classes, and provide them externally via Maven. But that created another problem that Jenkins Test Harness broke down because of this (as [this](https://www.jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/#pluginfirstclassloader-and-its-discontents) section had warned.)